### ISSUE TITLE
The search box should select the search text when the cursor is back on the search bar

### ISSUE THREAD

**Issue by adityaponnada** [ryanhugh/searchneu#66]
Right now it simply places the cursor to a position, but if it selects the search text, user can start typing directly.

**Comment by ryanhugh** [comment]
Seems to me like this would be a good idea if people usually start their subsequent queries from scratch rather than editing their existing queries. We should have enough data in amplitude to figure this out. Also, looks like most of the other search engines on the internet do it the same way that we do now, but the omnibar on chrome does it this new way. 

I might take a look over the next week or so and figure out if we want to do this.

**Comment by adityaponnada** [comment]
Good point. Just that course search is limited to one or two keywords and in all likelihood, users might want to just search from scratch. I am just stating my experience and do not really have the data to back it up. But a feature request to consider. Great work BTW!

**Comment by ryanhugh** [comment]
Thanks a lot for letting us know and awesome to hear you like the site! :)
